runner.go

```diff
@@ -246,8 +247,9 @@ func (r *runner) getTask(index int) *Task {
 	return r.runTask[index]
 }
 
-func (r *runner) startSpawning(spawnCount int, spawnRate float64, spawnCompleteFunc func()) {
+func (r *runner) startSpawning(spawnCount int, spawnRate float64, host string, spawnCompleteFunc func()) {
 	Events.Publish(EVENT_SPAWN, spawnCount, spawnRate)
+	Events.Publish(EVENT_SPAWN3, spawnCount, spawnRate, host)
```

Comment:
What about Events.Publish(EVENT_CONFIG, "host", host)?

Reply:
I have modified the code to publish the `boomer:config` event along with a map https://github.com/myzhan/boomer/pull/205/commits/f090ff999025de78cd54867743a977e1e2fbe23c .
Is this what you meant?

Reply:
yes, thanks